src/integration-test/tests/citizen/directionsQuestionnaire/pages/hearing-exceptional-circumstances.ts

```diff
@@ -0,0 +1,23 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const fields = {
+  reason: 'textarea[id="reason"]'
+}
+
+const buttons = {
+  submit: 'input[type=submit]'
+}
+
+export class HearingExceptionalCircumstancesPage {
+  chooseYes (): void {
+    I.checkOption('Yes')
+    I.click(buttons.submit)
+  }
+
+  chooseNo (reason: string): void {
```

Comment:
I don't think we need to pass in the text. hard code it to reduce the number of variables to keep track of

Reply:
done